index.js

```diff
@@ -18,15 +19,19 @@ const riptaApiBaseUrl = 'http://realtime.ripta.com:81/api/';
 const staticOptions = { index: 'index.htm' };
 const validApiTypes = ['tripupdates', 'vehiclepositions', 'servicealerts'];
 
-const fetchBaseApi = (type, callback) => {
+const fetchBaseApi = (type) => {
```

Comment:
I think this commit message is misleading. You're promisifying the base API requests. 

Also this function might be better stored in a model as well.